packages/abap-deploy-config-inquirer/src/prompts/questions/config/package.ts

```diff
@@ -128,17 +129,33 @@ export function getPackagePrompts(
                 input: string | ListChoiceOptions,
                 answers: AbapDeployConfigAnswersInternal
             ): Promise<boolean | string> => {
+                // Compare current answers with previous answers
+                if (prevAnswers != null) {
+                    const keys = Object.keys(answers) as (keyof AbapDeployConfigAnswersInternal)[];
+                    // Defensive: prevAnswers is defined, so prevAnswers[key] is safe
+                    const onlyDescriptionChanged =
+                        keys.every((key) => key === 'description' || prevAnswers![key] === answers[key]) &&
+                        prevAnswers!.description !== answers.description;
+                    if (onlyDescriptionChanged) {
+                        prevAnswers = { ...answers };
+                        return true;
```

Comment:
We'd also need to keep track of the previous validation result for `packageAutocomplete`. Returning true just because only the description has changed, could be a false positive. For `packageAutocomplete` this is highly unlikly if not impossible, but `packageManual` would need to be considered more